test_string_calculator.py

```diff
@@ -0,0 +1,43 @@
+from unittest import TestCase
+import unittest
+from string_calculator import StringCalculator
+
+
+class StringCalculatorTest(TestCase):
+    def setUp(self):
+        self.string_calculator = StringCalculator()
+
+    def test_string_calculator_zero(self):
```

Comment:
maybe the name of the method can be improved 
```test_string_calculator_empty_string_as_input```

Reply:
Done.
sometimes I'm not sure about large names in functions, that's why I tend to be kind of short with them.